addons/mass_mailing/static/src/js/mailing_contact_view_kanban.js

```diff
@@ -12,9 +12,12 @@ var MailingContactController = KanbanController.extend({
     }),
 
     _onImport() {
-        this.do_action('mass_mailing.mailing_contact_import_action', {
-            additional_context: this.renderer.state.context,
-        });
+        const context = this.renderer.state.context;
+        const actionParams = { additional_context: context };
+        if ((!context.default_mailing_list_ids) && (context.active_model === 'mailing.list')) {
```

Comment:
There seems to be extra unnecessary parenthesis.

Also, shouldn't you check that you have active_ids ?

Reply:
Thanks, I have removed the extra parenthesis.